data-plane/control-protocol/src/main/java/dev/knative/eventing/control/protocol/impl/SerializedEventBusRequester.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.knative.eventing.control.protocol.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.eventbus.DeliveryOptions;
+import io.vertx.core.eventbus.EventBus;
+import io.vertx.core.eventbus.Message;
+import java.util.ArrayDeque;
+import java.util.Map;
+import java.util.Queue;
+
+/**
+ * This class sends requests to event bus in order, waiting every time for the previous one to reply before sending the next one.
```

Comment:
The reason to have this logic is to "stay on the safe side" to begin experimenting. Control messages should take very very short amount of time to be handled.

I foresee how, in case we start having scalability problems and this becomes a significant bottleneck, we can have "smarter" logic to serialize requests, e.g. like you sad, reconcile requests for trigger B could run in parallel with reconcile requests for trigger A, but two requests for trigger A should be serialized. But I want this to become a problem before tackling it, and that's the reason why I made a separate class for it, more than including this logic straight in the `TCPControlServerVerticle`.